src/pages/en/core-concepts/framework-components.md

```diff
@@ -119,32 +119,93 @@ import MyVueComponent from '../components/MyVueComponent.vue';
 Only **Astro** components (`.astro`) can contain components from multiple frameworks.
 :::
 
+## Passing Children to Framework Components
+
+Inside of an Astro component, you **can** pass children to framework components. Each framework has its own patterns for how to reference these children: React, Preact, and Solid all use a special prop named `children`, while Svelte and Vue use the `<slot />` element.
+
+
+```astro
+// src/pages/MyAstroPage.astro
+---
+import MyReactSidebar from '../components/MyReactSidebar.jsx';
+---
+<MyReactSidebar>
+  <p>Here is a sidebar with some text and a button.</p>
+</MyReactSidebar>
+```
+
+Additionally, you can use [Named Slots](/en/core-concepts/astro-components/#named-slots) to group specific children together. 
+
+For React, Preact, and Solid these slots will be converted to a top-level prop. Slot names using `kebab-case` will be converted to `camelCase`.
+
+```astro
+// src/pages/MyAstroPage.astro
+---
+import MySidebar from '../components/MySidebar.jsx';
+---
+<MySidebar>
+  <h2 slot="title">Menu</h2>
+  <p>Here is a sidebar with some text and a button.</p>
+  <ul slot="social-links">
+    <li><a href="https://twitter.com/astrodotbuild">Twitter</a></li>
+    <li><a href="https://github.com/withastro">GitHub</a></li>
+  </ul>
+</MySidebar>
+```
+
+```jsx
+// src/components/MySidebar.jsx
+export default function MySidebar({ children, title, socialLinks }) {
+  return (
+    <aside>
+      <header>{title}</header>
+      <main>{children}</main>
+      <footer>{socialLinks}</footer>
+    </aside>
+  )
+}
+```
+
+For Svelte and Vue these slots can be referenced using a `<slot>` element with the `name` attribute. Slot names using `kebab-case` will be preserved.
+
+```jsx
+// src/components/MySidebar.svelte
+<aside>
+  <header><slot name="title" /></header>
+  <main><slot /></main>
+  <footer><slot name="social-links" /></footer>
+</aside>
+```
+
 ## Nesting Framework Components
 
-Inside of an Astro component, you can also nest components from multiple frameworks.
+Inside of an Astro file, component children can also be hydrated components. This means that you can recursively nest components from any framework.
```

Comment:
```suggestion
Inside of an Astro file, framework component children can also be hydrated components. This means that you can recursively nest components from any of these frameworks.
```
2 suggestions to consider:

1. We've been saying "framework component" or "UI framework component" to distinguish from Astro component (which is... not a framework? That's another discussion!)

2. I also added "of these" to imply "of the UI frameworks we support" ... not ANY framework.

Reply:
Great suggestion, thank you!